exercises/http_client/problem.fr.md

```diff
@@ -0,0 +1,44 @@
+Écrivez un programme qui fait une requête HTTP GET sur une URL fournie
+comme premier argument de la ligne de commande.  Affichez le contenu
+`String` de **chaque** événement 'data' de la réponse sur sa propre
+ligne dans la console (stdout).
+
+----------------------------------------------------------------------
+
+## CONSEILS
+
+Pour cet exercice, vous aurez besoin du module noyau `http`.
+
+La documentation de ce module peut être lue hors-ligne en navigant ici :
+
+  {rootdir:/node_apidoc/http.html}
+
+La méthode `http.get()` est un raccourci pour les requêtes GET simples,
+vous pouvez l’utiliser pour simplifier votre solution.  Le premier argument
+peut être l’URL que vous voulez récupérer ; passez une fonction de rappel
+en deuxième argument.
+
+Contrairement aux autres fonctions de rappel, celle-ci a comme signature :
+
+```js
+function callback (response) { /* ... */ }
+```
+
+…dans laquelle l’objet `response` est un objet **flux** Node.  Vous pouvez
```

Comment:
I'm not sure "stream" should translated, or maybe something like "**flux** (stream)" for the first occurrence ?